### PR TITLE
Jorge/wip

### PR DESCRIPTION
**Pull request by jorgefandinno**

### CODE REVIEW

**Comment by rkaminsk** [comment]
You do not have to close a pull request and open a new one if you make changes. Just commit into your branch; all your commits will accumulate in the pull request.

File: krr.bib [hunk 1]
```diff
@@ -20174,3 +20174,129 @@ @InProceedings{zlapop90
   editor =	 "K. Konolige",
   pages =	 "259-271"
 }
+
+@inproceedings{cafafa19a,
+  author =   "Cabalar, P. and Fandinno, J. and {Fari{\~{n}}as del Cerro}, L.",
```
Comment (tortinator) [inline krr.bib]: With us this would be

P. Cabalar and J. Fandinno and L. Farinas ...

Comment (jorgefandinno) [reply krr.bib]: I changed the authors' order.
How it looks now?

File: procs.bib [hunk 2]
```diff
@@ -3329,62 +3329,62 @@ @Proceedings{zue12
...
 
 @proceedings{epia17,
-    editor = "Oliveira, E. and Gama, J. and Vale, Z. and Lopes Cardoso, H.",
-    isbn = "978-3-319-65339-6",
-    year = "2017",
-    title = "Progress in Artificial Intelligence - Proceedings of the Eighteenth {EPIA} Conference on Artificial Intelligence {(EPIA'17)}",
-    series = "Lecture Notes in Computer Science",
-    volume = "10423",
-    publisher = "Springer"
+  editor =	 "E. Oliveira and J. Gama and Z. Vale and H. {Lopes Cardoso}",
```
Comment (tortinator) [inline procs.bib]: spaces

File: procs.bib [hunk 3]
```diff
@@ -3329,62 +3329,62 @@ @Proceedings{zue12
...
+  isbn = "978-3-319-65339-6",
```
Comment (tortinator) [inline procs.bib]: no isbn

File: procs.bib [hunk 4]
```diff
@@ -3329,62 +3329,62 @@ @Proceedings{zue12
...
 }
 
 @proceedings{aaai91,
-    editor = "Dean, T. and McKeown, K.",
-    isbn = "0-262-51059-6",
-    title = "Proceedings of the Ninth National Conference on Artificial Intelligence {(AAAI'91)}",
-    year = "1991",
-    publisher = "{AAAI} Press / The {MIT} Press"
+  editor =	 "T. Dean and K. McKeown",
+  isbn = "0-262-51059-6",
```
Comment (tortinator) [inline procs.bib]: no isbn

File: procs.bib [hunk 5]
```diff
@@ -3329,62 +3329,62 @@ @Proceedings{zue12
...
 
 @proceedings{iclp18,
-    editor = "{Dal Pal{\`{u}}}, A. and Tarau, P. and Saeedloei, N. and Fodor, P.",
-    year = "2018",
-    title = "Technical Communications of the Thirty-fourth International Conference on Logic Programming {(ICLP'18)}",
-    series = "{OASICS}",
-    isbn = "978-3-95977-090-3",
-    volume = "64",
-    publisher = "Schloss Dagstuhl - Leibniz-Zentrum fuer Informatik"
+  editor =	 "A. {Dal Pal{\`u}} and P. Tarau and N. Saeedloei and P. Fodor",
```
Comment (tortinator) [inline procs.bib]: spaces

File: procs.bib [hunk 6]
```diff
@@ -3329,62 +3329,62 @@ @Proceedings{zue12
...
 
 @proceedings{lics92,
-    isbn = "0-8186-2735-2",
-    title = "Proceedings of the Seventh Annual Symposium on Logic in Computer Science {(LICS'92)}",
-    year = "1992",
-    publisher = "{IEEE} Computer Society"
+  title =	 "Proceedings of the Seventh Annual Symposium on Logic in Computer Science (LICS'92)",
+  publisher =	 "{IEEE} Computer Society",
+  year =	 1992,
+  isbn =	 "0-8186-2735-2"
```
Comment (tortinator) [inline procs.bib]: no isbn

File: procs.bib [hunk 7]
```diff
@@ -3329,62 +3329,62 @@ @Proceedings{zue12
...
 }
 
 @proceedings{lpnmr91,
-    editor = "Nerode, A. and Wiktor Marek, V. and Subrahmanian, V.",
-    isbn = "0-262-64027-9",
-    title = "Proceedings of the Logic Programming and Non-monotonic Reasoning {(LPNMR'91)}",
-    year = "1991",
-    publisher = "The {MIT} Press"
+  editor =	 "A. Nerode and V. Marek and V. Subrahmanian",
```
Comment (tortinator) [inline procs.bib]: spaces

File: procs.bib [hunk 8]
```diff
@@ -3329,62 +3329,62 @@ @Proceedings{zue12
...
+  isbn = "0-262-64027-9",
```
Comment (tortinator) [inline procs.bib]: no isbn

File: procs.bib [hunk 9]
```diff
@@ -3329,62 +3329,62 @@ @Proceedings{zue12
...
 @proceedings{birthday11gelfond,
-    editor = "Balduccini, M. and Son, T.",
-    isbn = "978-3-642-20831-7",
-    year = "2011",
-    title = "Logic Programming, Knowledge Representation, and Nonmonotonic Reasoning - Essays Dedicated to Michael Gelfond on the Occasion of His Sixty-fifth Birthday",
-    series = "Lecture Notes in Computer Science",
-    volume = "6565",
-    publisher = "Springer"
+  editor =	 "M. Balduccini and T. Son",
+  isbn = "978-3-642-20831-7",
```
Comment (tortinator) [inline procs.bib]: no isbn

File: procs.bib [hunk 10]
```diff
@@ -3329,62 +3329,62 @@ @Proceedings{zue12
...
 }
 
 @proceedings{ijcai91,
-    editor = "Mylopoulos, J. and Reiter, R.",
-    isbn = "1-55860-160-0",
-    title = "Proceedings of the Twelfth International Joint Conference on Artificial Intelligence {(IJCAI'91)}",
-    year = "1991",
-    publisher = "Morgan Kaufmann"
+  editor =	 "J. Mylopoulos and R. Reiter",
+  isbn = "1-55860-160-0",
```
Comment (tortinator) [inline procs.bib]: no isbn

**Comment by jorgefandinno** [comment]
I make some changes. 
Though, to say the truth, I am a bit lost. Was this what you asked for.

Also, I did not find the editor of lics 92. They do not appear in DBLP. 
The organisers


General Chair: Robert L. Constable
Program Chair: Andre Scedrov
Conference Chair: Phokion G. Kolaitis
Publicity Chair: Daniel Leivant

should we add these as editors?

**Pull request closed without merging**